perfkitbenchmarker/configs/storage_utility.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+import numpy
+
+
+class SQLProviderError(Exception):
+  pass
+
+
+class IOPSError(Exception):
+  pass
+
+
+class StorageConfig(object):
```

Comment:
Let's go with DiskIopsToCapacity then. It expresses following: it takes in a disk iops number, then converts to "capacity". "Capacity" here means: size of disk, number of cores, and number of disk.